data/actions/scripts/other/potions.lua

```diff
@@ -1,29 +1,75 @@
-local ultimateHealthPot = 8473
-local greatHealthPot = 7591
-local greatManaPot = 7590
-local greatSpiritPot = 8472
-local strongHealthPot = 7588
-local strongManaPot = 7589
-local healthPot = 7618
-local manaPot = 7620
-local smallHealthPot = 8704
-local antidotePot = 8474
-local greatEmptyPot = 7635
-local strongEmptyPot = 7634
-local emptyPot = 7636
-
+potions = {
+	removePots = true,
+	addEmptyPots = true, -- Only works if removepots isn't false or nil
+	words = "Aaaah...",
+	-- Health & Spirit Potions
+	[8473] = { -- ultimate health potion
+		vocations = {4, 8},
+		minLevel = 130,
+		health = { min = 650, max = 850},
+		emptyPot = 7635
+	},
+	[7591] = { -- great health potion
+		vocations = {4, 8},
+		minLevel = 80,
+		health = { min = 425, max = 575},
+		emptyPot = 7635
+	},
+	[8472] = { -- great spitit potion
+		vocations = {3, 7},
+		minLevel = 80,
+		health = { min = 250, max = 350},
+		mana = { min = 100, max = 200},
+		emptyPot = 7635
+	},
+	[7588] = { -- strong health potion
+		vocations = {3, 4, 7, 8},
+		minLevel = 50,
+		health = { min = 250, max = 350},
+		emptyPot = 7634
+	},
+	[7618] = { -- health potion
+		health = { min = 125, max = 175},
+		emptyPot = 7636
+	},
+	-- Mana Potions
+	[7590] = { -- great mana potion
+		vocations = {1, 2, 5, 6},
+		minLevel = 80,
+		mana = { min = 150, max = 250},
+		emptyPot = 7635
+	},
+	[7589] = { -- strong mana potion
+		vocations = {1, 2, 3, 5, 6, 7},
+		minLevel = 50,
+		mana = { min = 115, max = 185},
+		emptyPot = 7634
+	},
+	[7620] = { -- mana potion
+		mana = { min = 75, max = 125},
+		emptyPot = 7636
+	},
+	[8704] = { -- small health potion
+		health = { min = 60, max = 90},
+		emptyPot = 7636
+	}
+}
+-- antidote
 local antidote = Combat()
 antidote:setParameter(COMBAT_PARAM_TYPE, COMBAT_HEALING)
 antidote:setParameter(COMBAT_PARAM_EFFECT, CONST_ME_MAGIC_BLUE)
 antidote:setParameter(COMBAT_PARAM_TARGETCASTERORTOPMOST, true)
 antidote:setParameter(COMBAT_PARAM_AGGRESSIVE, false)
 antidote:setParameter(COMBAT_PARAM_DISPEL, CONDITION_POISON)
+potions[8474] = {combat = antidote, emptyPot = 7636}
+
+--- FUNCTIONS ---
 
 local exhaust = Condition(CONDITION_EXHAUST_HEAL)
 exhaust:setParameter(CONDITION_PARAM_TICKS, (configManager.getNumber(configKeys.EX_ACTIONS_DELAY_INTERVAL) - 100))
```

Comment:
Actually, the user can edit it, as he prefer.
Check the last 2 commits, there is a lot of modifications.